srcpkgs/clthreads/template

```diff
@@ -0,0 +1,36 @@
+# Template file for 'clthreads'
+pkgname=clthreads
+version=2.4.2
+revision=1
+build_wrksrc="source"
+build_style=gnu-makefile
+make_install_args="LIBDIR=/usr/lib"
+makedepends="glibc-devel"
+short_desc="C++ wrapper library around the POSIX threads API"
+maintainer="bluntphenomena <gregwyd@gmail.com>"
+license="LGPL-2.1-or-later"
+homepage="https://kokkinizita.linuxaudio.org/linuxaudio/"
+distfiles="https://kokkinizita.linuxaudio.org/linuxaudio/downloads/${pkgname}-${version}.tar.bz2"
+checksum=c659b14c0c4055c44432cb83060b95d30ae0c1ecc6f50d73968e239c100f7a31
+
+pre_build() {
+        sed -i '/march=native/d' Makefile
```

Comment:
replce sed with vsed, it will then warn in the future if this workaround is no longer required